Cargo.toml

```diff
@@ -18,11 +18,12 @@ exclude = ["/.*"]
 async-channel = "1.4.0"
 async-lock = "2.6.0"
 async-task = "4.0.2"
-atomic-waker = "1.0.0"
 fastrand = "1.3.4"
 futures-io = { version = "0.3.28", default-features = false, features = ["std"] }
 futures-lite = { version = "1.11.0", default-features = false }
 log = "0.4.17"
+piper = "0.2.0"
 
 [dev-dependencies]
 futures-lite = "1.11.0"
+log = "0.4.17"
```

Comment:
only question: why this?

Reply:
Accidentally a line during the rebase. Fixed!